codegen/chainconfig.go

```diff
@@ -1,143 +1,47 @@
 package codegen
 
-type ChainConfig struct {
-	ID               string // Public
-	DisplayName      string // Public
-	ExplorerLink     string
-	ApiEndpoint      string
-	FirehoseEndpoint string
-	Network          string
-	SupportsCalls    bool
+import (
+	"log"
+	"sync"
+
+	registry "github.com/pinax-network/graph-networks-libs/packages/golang/lib"
+)
+
```

Comment:
Let's use custom type for registry:

```
// NetworkRegistry is a thin wrapper around a [map[string]*registry.Network] to add some helper methods.
type NetworkRegistry map[string]*registry.Network
```

That way we will be able to attach methods to it.